auth_session_timeout/models/ir_config_parameter.py

```diff
@@ -14,23 +14,34 @@ class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
     @tools.ormcache(skiparg=0)
-    def get_session_parameters(self, db):
+    def _auth_timeout_get_parameter_delay(self, db):
         param_model = self.pool['ir.config_parameter']
         cr = self.pool.cursor()
```

Comment:
I have taken the manual cursors out for now, and everything still *appears* to work correctly.

In order to use a manual cursor with the newer API, I will have to create a new env instance, from what I can see.